pga_socket.scad

```diff
@@ -4,7 +4,7 @@
 /*[Variant]*/
 
 // Which socket you need
-pga_which = "128"; // ["12","64","68-10x10":68 10x10 - MC68000,"68-11x11":68 11x11 - 80C286,"84","114","128":128 - MC68030,"132","176","223","238"]
+pga_which = "128"; // ["12","64","68-10x10":68 10x10 - MC68000,"68-11x11":68 11x11 - 80C286,"84","114","128":128 - MC68030,"132","179" - MC68040RCxx, "182" - MC68040RCxxV,"223","238"]
```

Comment:
Syntax is wrong there, you need to have `"value":description`, so `"179":179 - MC68040RCxx` and so on.

Reply:
Didn't realize it was more than a comment, should be fixed